### DISCUSSION TITLE
update

### DISCUSSION THREAD

**Discussion by Facuudelvaa** [MustardChef/WSABuilds#59] [General]
a query. at the time of updating. I should delete the folder where wsa is and copy the new one but it would delete my data, right?

**Comment by MustardChef** [comment]
Yes, do not delete the folder with the current build as you will lose your data. Copy the files from the .zip with the updated build into the folder with the current build, replace all files and folders when asked and then run Run.bat to complete the update.

**Comment by bensavio8** [comment]
ERROR ON EXTRACTING FILES

**Reply by MustardChef** [reply]
@bensavio8 
[![image](https://github.com/MustardChef/WSABuilds/assets/68516357/e73cb77d-099a-46cb-8898-6447b9c93616)](https://github.com/MustardChef/WSABuilds/blob/master/Documentation/Fix%20Guides/Pre-Install%20Issues/Extraction%20Errors.md)

**Reply by 2chiefk** [reply]
And I hope you got my messages man but all the issue is right now that I've been seeing is substrates when you're installing we're getting two of them and it shuts one off sometimes randomly in settings go to your settings for Windows substrate and just check it you'll see you have two and every once in awhile when the next time it doesn't work right check and you'll see one of the substrates is off and that's all it is let's see if we can figure out a way to lock that down
